src/sessions/video-session-handler.ts

```diff
@@ -387,6 +387,11 @@ export default class VideoSessionHandler extends BaseSessionHandler {
       if (!session.videoMuted) {
         await this.setVideoMute(session, { id: session.id, mute: true }, true);
       }
+
+      stream.getTracks().forEach((track: MediaStreamTrack) => {
+        track.addEventListener('ended', this.stopScreenShare.bind(this, session));
+      });
```

Comment:
Needed this function to handle screenshares where the user clicks "Stop Sharing" from the browser prompt.